tests/wrappers/test_autoreset.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+from typing import Optional
+
+import numpy as np
+
+import gym
+from gym.wrappers import AutoResetWrapper
+
+
+class DummyResetEnv(gym.Env):
+    metadata = {}
+
+    def __init__(self):
```

Comment:
Add a docstring describing what this environment does please

Reply:
done